src/jats/jats-exporter.ts

```diff
@@ -424,10 +433,14 @@ export class JATSExporter {
     }
   }
 
-  protected buildFront = (doi?: string, id?: string, links?: Links) => {
-    const manuscript = findManuscript(this.modelMap)
-    const titles = findTitles(this.modelMap)
-
+  protected buildFront = (
+    manuscript: Manuscript,
+    journal?: Journal,
+    doi?: string,
+    id?: string,
+    links?: Links
```

Comment:
Are these used? We should just use whatever is in the manuscript node.

Reply:
This also applies to all the options. Anything we don't use in pressroom we can drop.